### DISCUSSION TITLE
Welcome to candi Discussions!

### DISCUSSION THREAD

**Discussion by tjhei** [dealii/candi#302] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by prasadnewton** [comment]
can anyone tell me which command i have to use to install dealii 9.1.1 version??/
